Dockerfile

```diff
@@ -27,7 +27,18 @@ RUN cargo install --path forest
 # Prod image for forest binary
 FROM debian:buster-slim
 
+# Install binary dependencies
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y curl file gcc g++ hwloc libhwloc-dev make openssh-client \
+    autoconf automake cmake libtool libcurl4 libcurl4-openssl-dev libssl-dev \
+    libelf-dev libdw-dev binutils-dev zlib1g-dev libiberty-dev wget \
+    xz-utils pkg-config python clang ocl-icd-opencl-dev ca-certificates
+
 # Copy over binaries from the build-env
 COPY --from=build-env /usr/local/cargo/bin/forest /usr/local/bin/forest
 
+# Expose ports for RPC
+EXPOSE 1234
+EXPOSE 64899
```

Comment:
uh, isn't this the libp2p port that is randomly generated unless specified? Might want to make this deterministic if so

Reply:
wait you removed the port, but won't it be necessary to expose the libp2p port to use the node meaningfully? Why not expose an arbitrary port and just set that port in config with the command in the makefile to run?

Reply:
Container runs fine without exposing port, if someone does need to expose a port to call into the container they can do so via command line still with the port of their choosing.